certbot/main.py

```diff
@@ -482,6 +482,9 @@ def _init_le_client(config, authenticator, installer):
         #crypto_util.validate_key_csr(acc.key)
     else:
         acc, acme = None, None
+    if sys.version_info[:2] == (3, 3):
```

Comment:
Doing this here has the same problem as I mentioned before. People running `certbot renew` won't get notifications until their certificate is renewed. I think we should do this in `certbot/main.py` in `main()` sometime after the `try` `except` blocks around `log.post_arg_parse_setup`.